lib/honeycomb/integrations/active_support.rb

```diff
@@ -50,6 +50,13 @@ def default_handler
             # Make ActionController::Parameters parseable by libhoney.
             value = value.to_unsafe_hash if value.respond_to?(:to_unsafe_hash)
             span.add_field("#{name}.#{key}", value)
+
+            error, error_detail = payload[:exception]
```

Comment:
Aww yiss, array expansion. I think, though, that since you're grabbing the key off `payload` this ought to happen outside the `payload.each do … end` loop. The end result is correct, but this is extra work inside the loop.

Reply:
Derp...not sure why I didn't see that :+1: